spec/generate_rota_spec.rb

```diff
@@ -97,10 +98,6 @@
       roles_config,
     )
 
-    rota_generator.people.each do |p|
```

Comment:
I expect you intended to squash commits 2 and 3?

Reply:
Hmm - actually I was planning to keep them apart, I just lost track slightly (I would have made another commit with this remove and the name sort removed and squashed that into commit 2). I was going to say "maybe not too late to rebase!" but people have added new features, so it probably is.